<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>528</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>494</width>
    <height>528</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>494</width>
    <height>534</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Config dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>471</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="envTab">
    <attribute name="title">
     <string>Environment</string>
    </attribute>
    <widget class="QGroupBox" name="audioGroupBox">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>138</y>
       <width>441</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Audio</string>
     </property>
     <widget class="QWidget" name="layoutWidget3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>411</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="bgmVolumeLabel">
         <property name="text">
          <string>BGM    Volume</string>
         </property>
         <property name="buddy">
          <cstring>bgmVolumeSlider</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="bgmVolumeSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget3_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>411</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="effectVolumeLabel">
         <property name="text">
          <string>Effect Volume</string>
         </property>
         <property name="buddy">
          <cstring>bgmVolumeSlider</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="effectVolumeSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="enableBgMusicCheckBox">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>71</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable background music</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="randomPlayBGMCheckBox">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>90</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Random Play BGM</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>68</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="browseBgMusicButton">
         <property name="text">
          <string>Browse ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetBgMusicButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>23</y>
        <width>421</width>
        <height>40</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="setupBgMusicLabel">
         <property name="text">
          <string>Setup background music</string>
         </property>
         <property name="buddy">
          <cstring>bgMusicPathLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bgMusicPathLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>108</y>
        <width>324</width>
        <height>18</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="enableEffectCheckBox">
         <property name="text">
          <string>Enable skill and card effects</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableLastWordCheckBox">
         <property name="text">
          <string>Enable last word</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="bgGroupBox">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>8</y>
       <width>441</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Background</string>
     </property>
     <widget class="QWidget" name="layoutWidget4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>73</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="backgroundSetupLabel">
           <property name="text">
            <string>Setup main background</string>
           </property>
           <property name="buddy">
            <cstring>bgPathLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="bgPathLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="browseBgButton">
           <property name="text">
            <string>Browse ...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetBgButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="noIndicatorCheckBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>95</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No indicator</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="noEquipAnimCheckBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No equip anim</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="fullSkinCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Use full skin</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget4_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>201</width>
        <height>73</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="backgroundSetupLabel_2">
           <property name="text">
            <string>Setup game background</string>
           </property>
           <property name="buddy">
            <cstring>gameBgPathLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="gameBgPathLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="browseGameBgButton">
           <property name="text">
            <string>Browse ...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetGameBgButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="fontSetupGroupBox">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>338</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Font setup</string>
     </property>
     <widget class="QPushButton" name="changeAppFontButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set application font</string>
      </property>
     </widget>
     <widget class="QLabel" name="appFontLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>96</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Application font</string>
      </property>
      <property name="buddy">
       <cstring>appFontLineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="appFontLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>23</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditFontLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="setTextEditFontButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Font ...</string>
      </property>
     </widget>
     <widget class="QLabel" name="textEditFontLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>52</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Text edit font</string>
      </property>
      <property name="buddy">
       <cstring>textEditFontLineEdit</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="setTextEditColorButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Color ...</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="showMsgBoxWhenExitCheckBox">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>426</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Show confirm message box when exit</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gameTab">
    <attribute name="title">
     <string>Game</string>
    </attribute>
    <widget class="QCheckBox" name="neverNullifyMyTrickCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Never nullify my single target trick</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="autoTargetCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable auto target</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="intellectualSelectionCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable intellectual selection</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="doubleClickCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable double-click</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="delayLabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>91</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Delay</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="secondsSpinBox">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>91</y>
       <width>36</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLabel" name="autoCloseCardContainerLabel">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>91</y>
       <width>360</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>seconds, auto close card container that need close manually</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>115</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="bubbleChatBoxDelayLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Bubble chat box delay(seconds)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="bubbleChatBoxDelaySpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableBgMusicCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>randomPlayBGMCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
